recipes/thuner/meta.yaml

```diff
@@ -0,0 +1,82 @@
+{% set name = "thuner" %}
+{% set version = "0.0.10" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f8ae60837da55fe699f519751f69a60af45e39cb8cb1a7b0b7f31a3b66be5036
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
+    
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - graphviz =12.2.1
+    - python-graphviz =0.20.3
+    - pydot =3.0.4
+  run:
+    - python
+    - numpy =2.0.2
+    - numba =0.60.0
+    - scipy =1.14.1
+    - scikit-image =0.24.0
+    - pandas =2.2.3
+    - matplotlib-base >=3.9
+    - xarray =2024.9.0
+    - cartopy =0.24.0
```

Comment:
You should ~never pin exactly or your package will never be able to be installed in an environment alongside other packages.

I consider this a packaging bug either here or in the upstream package.